gateway/src/apicast/oauth/oidc.lua

```diff
@@ -51,7 +51,7 @@ function _M.new(service)
     jwt_claims = {
       -- 1. The JWT MUST contain an "iss" (issuer) claim that contains a
       -- unique identifier for the entity that issued the JWT.
-      iss = jwt_validators.equals_any_of({ issuer }),
+      iss = issuer and jwt_validators.equals_any_of({ issuer }) or jwt_validators.required(),
```

Comment:
Why is `or jwt_validators.required()` needed?

Reply:
Basically, I want to validate the issuer, if there is some, or just require the field.

Reply:
Hmm so is it OK if `iss` is there but not included in `issuer` ? :thinking:

Reply:
Well, that's the thing. If there is no issuer in the configuration, what to do?

I decided to just require any value for `iss`.